standard/interfaces.md

```diff
@@ -347,33 +347,23 @@ See §interface-static-constructors for information regarding the allocation and
 
 This clause augments the description of methods in classes [§15.6](classes.md#156-methods) for methods declared in interfaces.
 
-```ANTLR
-interface_method_header
-    : identifier '(' parameter_list? ')' ';'
-    | identifier type_parameter_list '(' parameter_list? ')'
-      type_parameter_constraints_clause* ';'
-    ;
-```
-
-The *attributes*, *return_type*, *ref_return_type*, *identifier*, and *parameter_list* of an interface method declaration have the same meaning as those of a method declaration in a class ([§15.6](classes.md#156-methods)).
-
-*method_modifier* shall not include `override`.
-
-An interface method declaration that has a block body or expression body as a *method_body* is a default implementation ([§18.1](interfaces.md#181-general)), so it is *not* abstract. A *method_declaration* shall not have *type_parameter_constraints_clause*s unless it also has a *type_parameter_list*.
-
-The list of requirements for valid combinations of modifiers stated for a class method is extended, as follows:
-
-- A static declaration that is not extern shall have a block body or expression body as a *method_body*.
-- A virtual declaration that is not extern shall have a block body or expression body as a *method_body*.
-- A private declaration that is not extern shall have a block body or expression body as a *method_body*.
-- A sealed declaration that is not extern shall have a block body or expression body as a *method_body*.
-- An async declaration shall have a block body or expression body as a *method_body*.
+Interface properties are declared using *method_declaration*s (§15.6)). The *attributes*, *return_type*, *ref_return_type*, *identifier*, and *parameter_list* of an interface method declaration have the same meaning as those of a method declaration in a class. Interface methods have the following additional rules:
```

Comment:
Also, do we really need to enumerate some of the syntactic parts and say their meaning doesn't change? If so, should we include the type parameter list?
```suggestion
Interface methods are declared using *method_declaration*s (§15.6)). The *attributes*, *return_type*, *ref_return_type*, *identifier*, and *parameter_list* of an interface method declaration have the same meaning as those of a method declaration in a class. Interface methods have the following additional rules:
```